src_assets/linux/misc/postinst

```diff
@@ -1,5 +1,11 @@
 #!/bin/sh
 
+# rpm-ostree based images have a read-only system, so the following commands will fail
+if grep -q "OSTREE" /etc/os-release; then
+  echo "WARNING: Atomic desktop operating system detected. Skipping post-install steps."
+  exit 0
+fi
```

Comment:
```suggestion
if grep -q "OSTREE" /etc/os-release; then
  echo "WARNING: Atomic desktop operating system detected."
fi
```

How about we set a variable here, and then only run the setcap if this variable is not set (along with the other current condition). I think the other command runs fine?

Also, this needs a rebase due to another change in this file.

Reply:
So, I don't think this change is needed any longer. I was able to install the latest nightly today without any issues https://github.com/LizardByte/Sunshine/releases/tag/v2024.822.131433. There were no errors thrown about read-only filesystem.

I'm okay to close this pull request since the issue has now been resolved

Reply:
But I think the postinst will still fail at the setcap step and then not run the remainder of the script?

Reply:
Surprisingly, it didn't fail. Even when I tested your PR build earlier, it didn't throw any errors during install. I don't know how `rpm-ostree` works under the hood, but somehow, your change https://github.com/LizardByte/Sunshine/pull/3050/files also fixed the postinst for some reason